src/index.js

```diff
@@ -115,23 +120,14 @@ function deconstruct (selector, styles, media) {
 
     if (type === 'number' || type === 'string') {
       decs.push(createDec(key, value))
-      continue
     } else if (Array.isArray(value)) {
-      forEach(value, val => {
-        decs.push(createDec(key, val))
-      })
-      continue
-    } else if (/^:/.test(key)) {
+      forEach(value, val => decs.push(createDec(key, val)))
+    } else if (key.charCodeAt(0) === 58) {
       forEach(deconstruct(selector + key, value, media), r => rules.push(r))
-      continue
-    } else if (/^@media/.test(key)) {
+    } else if (key.indexOf('@media') !== -1) {
```

Comment:
Good point. There may be something I'm missing here. @threepointone might know if this is safe.